add_user.rb

```diff
@@ -0,0 +1,54 @@
+require './student'
+require './teacher'
+
+module AddUser
+  def add_user
+    puts "\nPlease select a number to choose an option:"
+    puts '1 - Create a student'
+    puts '2 - Create a teacher'
+    puts '0 - Exit'
+    user = gets.chomp
+
+    case user
+    when '1'
+      clear
+      add_student
+    when '2'
+      clear
+      add_teacher
+    when '0'
+      exit
+    else
+      invalid_prompt
+      add_user
+    end
+  end
+
+  def add_student
+    print 'Age: '
+    gets.chomp
+
+    print 'Name: '
+    gets.chomp
+
+    print 'Has parent permission? [Y/N]: '
+    gets.chomp
+
+    permission.downcase == 'y' || permission.downcase == 'yes' || permission == ''
+
+    response('Student')
+  end
```

Comment:
When I try to create a student I get an error.
![image](https://user-images.githubusercontent.com/38088818/174101222-3120d8c1-45ba-4281-83d5-7024a3174a1b.png)

- Kindly look into this to make sure that I can create a student successfully.